website/templates/emails/new_public_project.html.mako

```diff
@@ -0,0 +1,30 @@
+## -*- coding: utf-8 -*-
+<%inherit file="notify_base.mako"/>
+<%def name="content()">
+    <div style="margin: 40px;">
+        <br>
+        Hello ${fullname},
+        <br><br>
+        Congratulations on making a public project on the Open Science Framework (OSF)! Now that your project “${project_title}" is public, you’ll be able to take advantage of more OSF features:
+        <ul>
+            <li><a href="${osf_url}${nid}/files/">The number of downloads of your files will be automatically logged for you</a></li>
```

Comment:
Sorry, I should have specified -- instead of constructing the url, you can use reverse url lookup. For example, `node.web_url_for('collect_file_trees', _absolute=True)` should give you the url to the project files page including the domain. 

My original suggestion was to pass those in as context vars, but you can just put them directly in the email template.